.release-please-manifest.json

```diff
@@ -1,5 +1,6 @@
 {
   "libs/hooks/open-telemetry": "5.0.0",
   "libs/providers/go-feature-flag": "0.5.0",
-  "libs/providers/flagd": "0.7.0"
+  "libs/providers/flagd": "0.7.0",
```

Comment:
I wonder if we should update this to `flagd-server`. That can be done in a follow up issue if you think it's a good idea.

Reply:
I was thinking about that, but I don't know... I'm inclined to say no, but I don't feel too strongly.